### ISSUE TITLE
Edit master/target folder while rotating

### ISSUE THREAD

**Issue by PhilZ-cwm6** [Sentaroh/SMBSync2#80]
(build 73, for my follow up)
Tested the last source, thank you, looks nearly done
There are still a few unsaved views in Edit master/target folder:

- **Archive view:**
  + highlighted items in image are lost/reset on rotation
<img width="324" alt="Screenshot_1589520414" src="https://user-images.githubusercontent.com/2428217/82018526-c6e5ff00-9685-11ea-80ff-8074c16f45c0.png">

- **Zip view**
  + password confirmation field is always filled with master pw on rotation. If you let it empty or partially filled while password field is completed: on rotation, the confirmation pw field is completed
  + select file zip is also reset: browse folders, select an existing zip, rotate and back to portrait: selection is gone, select again the target zip file, press Save, the selected file was not saved and target field is empty

confirmation password
<img width="324" alt="Screenshot_1589520804" src="https://user-images.githubusercontent.com/2428217/82018920-8cc92d00-9686-11ea-9878-44ed6273d485.png">

Select Zip file
<img width="324" alt="Screenshot_1589521103" src="https://user-images.githubusercontent.com/2428217/82018922-8d61c380-9686-11ea-8d7a-7d163fa7fae4.png">

After rotation: it is unselected and can no longer be saved when selected
<img width="324" alt="Screenshot_1589521369" src="https://user-images.githubusercontent.com/2428217/82018924-8dfa5a00-9686-11ea-8f23-0452ffff705e.png">

After it was reselected again and save: target is empty
<img width="324" alt="Screenshot_1589521133" src="https://user-images.githubusercontent.com/2428217/82018925-8dfa5a00-9686-11ea-9ff5-1c7735dcc48b.png">

- **Folder path not saved and reset**
  + Select List Folders and browse into folders by clicking the directories
  + rotate device and back to portrait
  + press Select
  + folder shown in browser was not saved
<img width="324" alt="Screenshot_1589520969" src="https://user-images.githubusercontent.com/2428217/82019093-e3366b80-9686-11ea-9ce2-df8fb3e1307d.png">
<img width="324" alt="Screenshot_1589521016" src="https://user-images.githubusercontent.com/2428217/82019094-e4679880-9686-11ea-8a87-54142939f987.png">

- **Archived extensions view in landscape**
<img width="400" alt="Screenshot_1589519729" src="https://user-images.githubusercontent.com/2428217/82020023-9784c180-9688-11ea-88fa-6157500034e5.png">

Better to put on separate line even for large devices
Look at this commit which also includes a few margins fixes and remove non adapted devider and use the ZIP_DEFAULT entries
https://github.com/Sentaroh/SMBSync2/pull/78

**Comment by Sentaroh** [comment]
The problem has been fixed. Please check it out.

**Comment by PhilZ-cwm6** [comment]
Thank you, I tested, we're approaching :-)
(build 75 for my followup)

- **Options are saved but only after the first start of app: ZIP, Archive and all other views**
  + exit app
  + start app
  + create new task or edit existing one
  + edit target folder and change all settings
  + rotate
  + back to portrait
  + all fields, checkboxes and spinners are reset, same if you browse folder using List Folders
  + after that and during all session, it works

<img width="324" alt="Screenshot_1589543923" src="https://user-images.githubusercontent.com/2428217/82050109-d67e3b80-96b7-11ea-894f-b5d5259a3eae.png">


- **Target folder choice is not saved**
  + On create new task, master and target are the same
  + change target folder
  + cannot save because warning master and target are the same despite they are not !

<img width="324" alt="Screenshot_1589544795" src="https://user-images.githubusercontent.com/2428217/82049407-91a5d500-96b6-11ea-94ac-1a5447a365e4.png">

- **double slash in path names**
<img width="324" alt="Screenshot_1589543893" src="https://user-images.githubusercontent.com/2428217/82049453-9ff3f100-96b6-11ea-9672-f09aa1544bb1.png">

**Comment by Sentaroh** [comment]
Thank you for your testing.

I can't reproduce your Issue("Options are saved but only after the first start of app: ZIP, Archive and all other views").
1.Initial screen
![device-2020-05-16-125929](https://user-images.githubusercontent.com/4037866/82110021-1efc2e80-9776-11ea-8f37-0a8ac09f9303.png)
2.Change all options.
![device-2020-05-16-130044](https://user-images.githubusercontent.com/4037866/82110023-20c5f200-9776-11ea-8a0f-956858844251.png)
3.Rotate screen to landscape.
![device-2020-05-16-130106](https://user-images.githubusercontent.com/4037866/82110025-215e8880-9776-11ea-8a2a-937ddf8c6eb4.png)
4.Rotate screen to portlate
![device-2020-05-16-130125](https://user-images.githubusercontent.com/4037866/82110026-215e8880-9776-11ea-944c-f16ae700fc70.png)
5.After save
![device-2020-05-16-130149](https://user-images.githubusercontent.com/4037866/82110027-21f71f00-9776-11ea-9d89-27d3c7ee7d7f.png)
6.After save
![device-2020-05-16-130209](https://user-images.githubusercontent.com/4037866/82110028-228fb580-9776-11ea-8a00-f62992b39079.png)

I am attaching the APK I tested, so please test it.

[SMBSync2_debug.zip](https://github.com/Sentaroh/SMBSync2/files/4637800/SMBSync2_debug.zip)

Best regards.

**Comment by PhilZ-cwm6** [comment]
Thank you a lot
After a clean and delete build folder it worked properly 👍 
All issues are fixed, just a small consistency in layout I will open an issue
Best regards
